files/move_base_app/jni/Android.mk.in

```diff
@@ -8,6 +8,15 @@ LOCAL_C_INCLUDES := $(LOCAL_PATH)/include
 LOCAL_LDLIBS := -landroid
 LOCAL_STATIC_LIBRARIES := android_native_app_glue roscpp_android_ndk
 
+### Please customize these values appropriately for your ROS ecosystem
+
+ROS_MASTER_URI ?= http://192.168.1.100:11311 # defaults to environment variable if defined
+ROS_ANDROID_IP := 192.168.1.101 # MUST be device IP
+
+### End customization region
+
+LOCAL_CFLAGS := -DROS_MASTER_URI="__master:=$(ROS_MASTER_URI)" -DROS_ANDROID_IP="__ip:=$(ROS_ANDROID_IP)"
```

Comment:
Missing single quotes to retain the double quotes on the literal string:
`LOCAL_CFLAGS := -DROS_MASTER_URI='"__master:=$(ROS_MASTER_URI)"' -DROS_ANDROID_IP='"__ip:=$(ROS_ANDROID_IP)"'`
Please fix this and submit it again so it can be merged.